### ISSUE TITLE
Using `--fallback-language` with JSON

### ISSUE THREAD

**Issue by kraklin** [andreasewering/travelm-agency#151]
Hey, I'm trying to use `--fallback-language` with JSON, but I get `THIS SHOULD NOT HAPPEN` string instead of fallback one. I have tried to scan through the code and documentation and even when the docs say 

```
To disable missing key detection and default missing keys to some other of your languages, you can use { "--fallback-language": "{your language key, i.e. 'en'}" } as a top-level key.
``` 

I was not able to find it out in the code as a part of JSON decodings at all. Nor the example of it. My languages JSON files looks like this:

```
{
  "--fallback-language": "en",
  "poweredBy": "Betrieben von",
  "logoAlt": "Logo",
  ...
}
```

Is that correct way to specify that I would like to fallback to `en` language in JSON?

**Comment by andreasewering** [comment]
Hi @kraklin , thanks for noticing the error! This is actually visible in the interactive tutorial as well (https://andreasewering.github.io/travelm-agency/consistency). I'll try to fix this asap.

**Comment by kraklin** [comment]
@andreasewering - cool, I was not sure whether I understood where to put that fallback part and if I'm doing it correctly or not.

Looking forward for the fix ;)

**Comment by andreasewering** [comment]
Should be fixed now in 3.3.1.
The issue was the recent movement from records for each language towards seperate functions matching on the language, where I forgot about the fallback resolving.
The fallbacks are now resolved in a central location before any code generation happens, plus I added a regression test :)
